src/App.js

```diff
@@ -66,7 +68,7 @@ let ERC20VENDOR
 let ERC20IMAGE
 let ERC20NAME
 let LOADERIMAGE = burnerlogo
-let HARDCODEVIEW// = "loader"// = "receipt"
+let HARDCODEVIEW = "yourmodule"// = "loader"// = "receipt"
```

Comment:
Can we change HARDCODEVIEW to be `undefined` again?

Reply:
undefined will return unknown view. Had to default it to main to load the main view